src/main/java/com/happeningnow/happeningnow/Application.java

```diff
@@ -2,19 +2,12 @@
 
```

Comment:
I just realized that the package name is wrong since it has twice `happeningnow`

Reply:
ok